soominheo3268-seoul.md

```diff
@@ -0,0 +1,10 @@
+## Seoul, South Korea
+
+#### Places to eat
+  - Try some authentic Korean BBQ
```

Comment:
@hectorsector Definitely! Gaonnuri, Jongro BBQ, Baekjeong at Ktown (32th street & between 5th and Broadway)